### DISCUSSION TITLE
Does SCC count historical lines as well?

### DISCUSSION THREAD

**Discussion by xxfogs** [boyter/scc#461] [Q&A]
Does SCC count lines from previous git revisions that are now deleted or changed?

**Comment by boyter** [comment]
No. It only counts what is on the file system at the point you run it. I have seen people script a loop over every commit and save the output (SQL output would work well for this!) in order to get this information but as of now there is no way in the tool to do it.

If you do come up with something reusable please provide details and it can be added to the README or perhaps a wiki.

**Reply by xxfogs** [reply]
Thanks! Looping over every commit might be something I would try, but it sounds like a horror performance-wise.
